Gemfile.lock

```diff
@@ -348,3 +346,6 @@ DEPENDENCIES
   watir-webdriver
   web-console (~> 2.0)
   whenever
+
+BUNDLED WITH
```

Comment:
You want `bundle update`. But I think that will still write the "BUNDLED WITH" bit. There might be a flag you can pass to bundler to make it not do that, but my current solution has been to just `git add -p Gemfile.lock` and not include that chunk.